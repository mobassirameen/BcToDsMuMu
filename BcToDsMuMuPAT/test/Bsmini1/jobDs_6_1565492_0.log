000 (1565492.000.000) 04/06 13:40:37 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_1694_8585_240>
...
040 (1565492.000.000) 04/06 13:43:27 Started transferring input files
	Transferring to host: <128.142.64.161:9618?addrs=128.142.64.161-9618&noUDP&sock=3268_59fe_138402>
...
040 (1565492.000.000) 04/06 13:43:27 Finished transferring input files
...
001 (1565492.000.000) 04/06 13:43:28 Job executing on host: <128.142.64.161:9618?addrs=128.142.64.161-9618&noUDP&sock=2174_c2fb_3>
...
006 (1565492.000.000) 04/06 13:43:32 Image size of job updated: 12732
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (1565492.000.000) 04/06 13:43:32 Started transferring output files
...
040 (1565492.000.000) 04/06 13:43:32 Finished transferring output files
...
005 (1565492.000.000) 04/06 13:43:33 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	25648  -  Run Bytes Sent By Job
	16042  -  Run Bytes Received By Job
	25648  -  Total Bytes Sent By Job
	16042  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        4         4 
	   Disk (KB)            :      538    15000    264298 
	   Memory (MB)          :        0     8000      8000 
...
