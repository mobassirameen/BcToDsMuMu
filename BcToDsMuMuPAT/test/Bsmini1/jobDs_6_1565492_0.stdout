========================
====> SGE  wrapper <====
========================
--> Running SGE digi-reco job wrapper
file:root://se01.indiacms.res.in:1094//cms/store/user/digupta/BctoDsMuMu_MC_MINIAOD_5k/6
file:BctoDsMuMu_MC_.root
--> Environment
Tue Apr  6 13:43:28 CEST 2021
b7s09n7171.cern.ch
Linux b7s09n7171.cern.ch 3.10.0-1127.19.1.el7.x86_64 #1 SMP Tue Aug 25 17:23:54 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
Filesystem                     1K-blocks           Used      Available Use% Mounted on
/dev/vda1                       52417516       39851800       12565716  77% /
devtmpfs                        22525888              0       22525888   0% /dev
tmpfs                           22553028           6424       22546604   1% /dev/shm
tmpfs                           22553028              0       22553028   0% /sys/fs/cgroup
tmpfs                           22553028           3748       22549280   1% /run
tmpfs                            4510608              0        4510608   0% /run/user/0
/dev/mapper/poolvg-poollv      330143004       49052460      264297152  16% /pool
AFS                           2147483647              0     2147483647   0% /afs
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cvmfs-config.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/sft.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/atlas.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/lhcb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/dune.opensciencegrid.org
cvmfs2                          33013760       32941314          72446 100% /cvmfs/larsoft.opensciencegrid.org
cvmfs2                          33013760       32941314          72446 100% /cvmfs/alice-nightlies.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/atlas-online-nightlies.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/atlas-pixel-daq.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cms-bril.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cms-ci.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cms-opendata-conddb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/compass.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/compass-condb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/compass-mc.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/lhcbdev.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/na62.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/sft-nightlies.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/alice-ocdb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/alice.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/ams.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/atlas-condb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/atlas-nightlies.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cms-ib.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cms.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/geant4.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/ilc.desy.de
cvmfs2                          33013760       32941314          72446 100% /cvmfs/lhcb-condb.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/na61.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/fermilab.opensciencegrid.org
cvmfs2                           1024000         289881         734119  29% /cvmfs/dune.osgstorage.org
cvmfs2                          33013760       32941314          72446 100% /cvmfs/cernvm-prod.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/unpacked.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/ship.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/dampe.cern.ch
cvmfs2                          33013760       32941314          72446 100% /cvmfs/dirac.egi.eu
cvmfs2                          33013760       32941314          72446 100% /cvmfs/singularity.opensciencegrid.org
cvmfs2                          33013760       32941314          72446 100% /cvmfs/oasis.opensciencegrid.org
cvmfs2                          33013760       32941314          72446 100% /cvmfs/grid.cern.ch
home-j                     3302194302768  1847161420976  1455032881792  56% /eos/home-j
project-o                  1841406085672   923365760648   918040325024  51% /eos/project-o
home-t                     2215425514544  1264550254304   950875260240  58% /eos/home-t
cms                       64374846299100 41410314482920 22964531816180  65% /eos/cms
home-l                     2215425514544  1264550254304   950875260240  58% /eos/home-l
home-g                     3302194302768  1847161420976  1455032881792  56% /eos/home-g
home-a                     3302194302768  1847161420976  1455032881792  56% /eos/home-a
experiment                57192306866356 42648809250760 14543497615596  75% /eos/experiment
home-v                     2986232785472  1331770651756  1654462133716  45% /eos/home-v
home-m                     2986232785472  1331770651756  1654462133716  45% /eos/home-m
home-b                     2986232785472  1331770651756  1654462133716  45% /eos/home-b
home-c                     4167252018880  1154514386448  3012737632432  28% /eos/home-c
home-d                     2215425514544  1264550254304   950875260240  58% /eos/home-d
home-s                     2737813750472  1364112334252  1373701416220  50% /eos/home-s
home-f                     4167252018880  1154514386448  3012737632432  28% /eos/home-f
home-k                     3302194302768  1847161420976  1455032881792  56% /eos/home-k
home-h                     2737813750472  1364112334252  1373701416220  50% /eos/home-h
home-y                     2737813750472  1364112334252  1373701416220  50% /eos/home-y
home-z                     2215425514544  1264550254304   950875260240  58% /eos/home-z
home-e                     2986232785472  1331770651756  1654462133716  45% /eos/home-e
home-r                     2737813750472  1364112334252  1373701416220  50% /eos/home-r
project-m                  2286743569696  1391291392012   895452177684  61% /eos/project-m
-> which edg-gridftp-ls
/bin/edg-gridftp-ls
-> which globus-url-copy
/bin/globus-url-copy
-> which srmcp
/pool/condor/dir_21047
--> End of env testing
--> Setup CMSSW
/pool/condor/dir_21047
Tue Apr  6 13:43:29 CEST 2021
/pool/condor/dir_21047
Tue Apr  6 13:43:29 CEST 2021
total 60
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 15051 Apr  6 13:43 digupta.cc
-rw-r--r--.  1 digupta zh  7351 Apr  6 13:43 _condor_stdout
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh   546 Apr  6 13:43 _condor_stderr
/pool/condor/dir_21047
--> Run cmsRun
/pool/condor/dir_21047
Tue Apr  6 13:43:32 CEST 2021
cmsRun step1.py 
condor_exec.exe: line 85: cmsRun: command not found
Tue Apr  6 13:43:32 CEST 2021
/pool/condor/dir_21047
total 64
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 15051 Apr  6 13:43 digupta.cc
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh  1182 Apr  6 13:43 _condor_stderr
-rw-r--r--.  1 digupta zh    52 Apr  6 13:43 step1.log
-rw-r--r--.  1 digupta zh  8095 Apr  6 13:43 _condor_stdout
total 68
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 15051 Apr  6 13:43 digupta.cc
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh  1182 Apr  6 13:43 _condor_stderr
-rw-r--r--.  1 digupta zh    52 Apr  6 13:43 step1.log
-rw-r--r--.  1 digupta zh  8681 Apr  6 13:43 _condor_stdout
run: This is the end, my friend
